### PR TITLE
split functions from nlbin

### PR DESCRIPTION
**Pull request by ZhangXinNan**
process1 in nlbin is too long, I split some functions from process1.

### CODE REVIEW

File: ocropus-nlbin [hunk 1]
```diff
@@ -83,18 +83,94 @@ def dshow(image,info):
...
     if args.parallel<2: print_info("=== %s %-3d" % (fname, i))
     raw = ocrolib.read_image_gray(fname)
     dshow(raw,"input")
     # perform image normalization
-    image = raw-amin(raw)
-    if amax(image)==amin(image):
-        print_info("# image is empty: %s" % (fname))
-        return
-    image /= amax(image)
+    image = normalze(raw)
```
Comment (zuphilip) [inline ocropus-nlbin]: Typo?

**Comment by kba** [comment]
Makes sense to split up, easier to follow. Maybe the method names could be less generic (`binary`, `normalize`).

Need to test this, travis complains about undefined variables.

**Comment by ZhangXinNan** [comment]
@kba Could you give a better name to 'normalize' ?

**Comment by kba** [comment]
I renamed it `normalize_raw_image`, will merge once https://travis-ci.org/tmbdev/ocropy/builds/281275240 succeeds.

**Comment by kba** [comment]
Thanks, merged!

**Pull request closed without merging**